main.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/containership/go-containership.cloud/client"
+	"github.com/drone/drone-go/drone"
+	"github.com/drone/drone-go/plugin"
+)
+
+var (
+	buildDate string
+)
+
+func main() {
+	fmt.Printf("Drone ContainerShip Plugin built at %s\n", buildDate)
+
+	repo := drone.Repo{}
+	vargs := Params{}
+
+	plugin.Param("repo", &repo)
+	plugin.Param("vargs", &vargs)
+	plugin.MustParse()
+
+	if vargs.Application == "" {
+		vargs.Application = repo.Name
+	}
+
+	if vargs.ApiKey == "" {
+		fmt.Println("Error: ContainerShip Cloud API Key is required!")
+		os.Exit(1)
+	}
+
+	if vargs.ClusterId == "" {
+		fmt.Println("Error: ContainerShip Cloud Cluster ID is required!")
+		os.Exit(1)
+	}
+
+	if vargs.Image == "" {
+		vargs.Image = repo.FullName + ":latest"
```

Comment:
Just some personal taste but I prefer to move the default assignments to the top behind `plugin.MustParse()`